make/test.mk

```diff
@@ -87,6 +87,7 @@ e2e-migration-verify:
 	@echo "Migration tests successfully finished"
 
 .PHONY: e2e-deploy-latest
+e2e-deploy-latest: INSTALL_OPERATOR=true
```

Comment:
Just a question but do we want to do the same as https://github.com/codeready-toolchain/toolchain-e2e/pull/1218 ?